quelpa.el

```diff
@@ -371,9 +422,14 @@ If there is an error and no existing checkout return nil."
 (defun quelpa-get-melpa-recipe (name)
   "Read recipe with NAME for melpa git checkout.
 Return the recipe if it exists, otherwise nil."
-  (let* ((recipes-path (expand-file-name "package-build/recipes" quelpa-build-dir))
+  (let* ((recipes-path (car quelpa-melpa-recipe-dirs))
          (files (directory-files recipes-path nil "^[^\.]+"))
          (file (assoc-string name files)))
```

Comment:
I think it's easy to put it all under the `when` line. But then `file` and `recipes-path` are needed for `insert-file-contents-literally` line... how to do that without `let`?